requirements.txt

```diff
@@ -69,6 +69,9 @@ ftfy
 # To use the BERT model
 pytorch-pretrained-bert==0.4.0
 
+# For checkpointer (if python < 3.7)
+dataclasses
```

Comment:
please don't casually introduce new dependencies

Reply:
Sure. Did not mean to be casual about it! :-)
Dataclasses are a built in feature of python 3.7, so I thought it would be safe to use. But it's easy for me to change the code to not use it. Would that be preferred?

Reply:
as of right now allennlp only requires Python 3.6, so it's not safe to use 3.7-only features. it would be better not to use dataclasses.